### ISSUE TITLE
Doesn't work (Rider)

### ISSUE THREAD

**Issue by Leon99** [cezarypiatek/MappingGeneratorIssueTracker#2]
I added `<PackageReference Include="MappingGenerator" Version="2021.7.20" />` to my csproj, got "Generate mapping code" and "Generate mapping code using member functions" items in the quick fix popup, but neither of them does anything. Restarting Rider didn't help. Any ideas much appreciated.
Rider 2021.2, .NET Core 3.1. Code:
```
        private static Foo Map(Model.Foo f)
        {
        }
```

**Comment by cezarypiatek** [comment]
I need to check it. I will let you know today's (my) evening.

**Comment by cezarypiatek** [comment]
I can confirm that there's a problem on Rider and I managed to find the culprit. I will try to release fix ASAP.  
Many thanks for reporting this issue.

**Comment by cezarypiatek** [comment]
I've just released v2021.7.22 which should contain a fix. Would you mind checking it and letting me know if it's working for you?

**Comment by Leon99** [comment]
Thanks for the prompt reaction @cezarypiatek, much appreciated. I confirm it's fixed in the latest version.
